OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -296,11 +296,15 @@ public interface IFacingInfo : ITraitInfoInterface { int GetInitialFacing(); }
 
 	public interface ICrushable
 	{
-		void OnCrush(Actor crusher);
-		void WarnCrush(Actor crusher);
 		bool CrushableBy(HashSet<string> crushClasses, Player owner);
```

Comment:
It might be worth changing this to `bool CrushableBy(Actor self, Actor crusher, HashSet<string> crushClasses);` to match `INotifyCrushed`.